### PR TITLE
Awesome books

### PR DESCRIPTION
**Pull request by prolajumokeoni**
In this project: 
- Created a collection that keeps a list of books
- Created a function to add and remove with title and author
- Created a function to remove a book from the collection
-  Users can add books
- Displayed all books created

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  };
+  book.id = Math.floor(Math.random() * (100 - 1 + 1)) + 1;
+  retrievedBooks.push(book);
+  localStorage.setItem('books', JSON.stringify(retrievedBooks));
+  document.querySelector('#title').value = '';
+  document.querySelector('#author').value = '';
+  const parentDiv = document.querySelector('.books');
+  const div = document.createElement('div');
+  div.classList.add('book');
+  div.dataset.id = book.id;
```
Comment (Davidosky007) [inline index.js]: * Kindly fix the errors in the console this error shows when i click the add button 
image
* HINT 💭 : check the `retrievedBooks` variable as you can see the errors are coming from `line`9,32

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @Kalunge @prolajumokeoni 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
cr2 gif_almost-there


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N.A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.js (hunk 1 above)
Comment (Kalunge) [reply index.js]: Thanks for your review, We have corrected the error

**Review by Lameck1 — APPROVED** [review]
Hi, @prolajumokeoni & @Kalunge

You've done well addressing the concern from the previous review. Consider this task complete!  It's time to merge it :shipit: 

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**